chloggen/internal/chlog/entry.go

```diff
@@ -82,7 +120,11 @@ func (e Entry) String() string {
 	issueStr := strings.Join(issueStrs, ", ")
 
 	var sb strings.Builder
-	sb.WriteString(fmt.Sprintf("- `%s`: %s (%s)", e.Component, e.Note, issueStr))
+	component := e.Component
```

Comment:
I would rather rename it to something like `prefix`

Reply:
Not quite sure I follow, this is not used as a prefix, it's still used as the component name if provided.